Applications/WebEditor.qmd

```diff
@@ -36,33 +36,33 @@ Upon initial access to the provided link, users are presented with the following
 
 2. **Help**
     
-    The *Help* icon at the top of the screen will provide you with a short tour of the main section of the editor. 
+    The *Help* icon at the top of the screen provides a short tour of the main sections of the editor. 
 
 3. **Wizard**
 
-    The *Wizard* is an experimental feature that will help you to create openEO processes in a simple way for some common use cases.
+    The *Wizard* is an experimental feature designed to simplify the creation of openEO processes for common use cases.
```

Comment:
is it experimental?

Reply:
Yes, indeed, the wizard is experimental at the moment.